core/modules/filter/filter.install

```diff
@@ -538,8 +538,8 @@ function filter_update_1004() {
         'multiple' => FALSE,
         'remember' => 0,
         'default_group' => 'All',
-        'default_group_multiple' => [],
-        'group_items' => []
+        'default_group_multiple' => array(),
+        'group_items' => array()
```

Comment:
Can you add a trailing comma @klonos?

```suggestion
        'group_items' => array(),
```

Reply:
Good catch @BWPanda 👍🏼  ...that code must have been copy/pasted almost verbatim from a .json exported file, so both those `[]` as well as this missing comma after the last array element are a result of that.